deployment/src/main/java/io/quarkus/tika/deployment/devui/TikaDevUIProcessor.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.tika.deployment.devui;
+
+import org.apache.tika.detect.Detector;
+
+import io.quarkus.deployment.IsDevelopment;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.devui.spi.page.CardPageBuildItem;
+import io.quarkus.devui.spi.page.Page;
+import io.quarkus.devui.spi.page.PageBuilder;
+
+/**
+ * Dev UI card for displaying important details such as the Tika library version.
+ */
+public class TikaDevUIProcessor {
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    void createVersion(BuildProducer<CardPageBuildItem> cardPageBuildItemBuildProducer) {
+        final CardPageBuildItem card = new CardPageBuildItem();
+
+        final PageBuilder versionPage = Page.externalPageBuilder("Version")
```

Comment:
Maybe also mention that it's the library version, to avoid confusion with the extension

```suggestion
        final PageBuilder versionPage = Page.externalPageBuilder("Tika Version")
```